src/SymfonyCache/CustomTtlListener.php

```diff
@@ -33,6 +33,11 @@ class CustomTtlListener implements EventSubscriberInterface
      */
     private $keepTtlHeader;
 
+    /**
+     * @var bool
+     */
+    private $disableFallback;
```

Comment:
negative formulations can be confusing, i prefer a positive formulation

```suggestion
    private $fallbackToSmaxage;
```